sklearn/linear_model/ransac.py

```diff
@@ -278,10 +319,15 @@ def fit(self, X, y):
 
             # residuals of all data for current random sample model
             y_pred = base_estimator.predict(X)
-            diff = y_pred - y
-            if diff.ndim == 1:
-                diff = diff.reshape(-1, 1)
-            residuals_subset = residual_metric(diff)
+
+            # XXX: Remove this if block in 0.20
```

Comment:
I am not sure about the conventions in scikit-learn, but is this documented somehwere?